tracer/test/Datadog.Trace.Tests/Debugger/SymbolsTests/SymbolExtractor/Approvals/SymbolExtractorTest.Datadog.Trace.Tests.Datadog.Trace.Tests.Debugger.SymbolsTests.TestSamples.PartialClass.verified.txt

```diff
@@ -0,0 +1,134 @@
+﻿{
+  "service": "test",
+  "env": "test",
+  "version": "0",
+  "language": "dotnet",
+  "scopes": [
+    {
+      "scope_type": "assembly",
+      "name": "Datadog.Trace.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
+      "start_line": 0,
+      "end_line": 0,
+      "scopes": [
+        {
+          "scope_type": "class",
+          "name": "Datadog.Trace.Tests.Debugger.SymbolsTests.TestSamples.PartialClass",
+          "start_line": 14,
+          "end_line": 26,
+          "language_specifics": {
+            "super_classes": [
+              "System.Object"
+            ]
+          },
+          "symbols": [
+            {
+              "name": "_s",
+              "type": "String",
+              "symbol_type": "Field",
+              "line": -1
+            },
+            {
+              "name": "_s2",
+              "type": "String",
+              "symbol_type": "Field",
+              "line": -1
+            }
+          ],
+          "scopes": [
+            {
+              "scope_type": "method",
+              "name": ".ctor",
```

Comment:
@OmerRaviv @dudikeleti: As I already mentioned in another thread, I think we should rename `.ctor` and `.cctor` to the class name (and mention that the symbol represents a static/non-static constructor).
`.ctor` and `.cctor` will for sure confuse our customers.
WDYT?

Reply:
I don't have a strong opinion here because I don't know what might confuse more. in R# you can write `ctor`, `cctor`, `new` and class name to find the constructor of the class. For me class name is les intuitive but it is very subjective.
@OmerRaviv

Reply:
We decided to keep it as it is for now and revisit it later.
Resolving.